### PR TITLE
Feature/generate project nav

### PR DESCRIPTION
**Pull request by Tom-Camp**
Adding Project nav via a function in the View rather than gathering from disparate values and moving templates from Branding into the Blueprint repo.

https://jiraent.cms.gov/browse/ISPGBSS-555

This PR is dependent on changes that were made in [PR#27](https://github.com/CMSgov/rato-grq-branding/pull/27)

### CODE REVIEW

**Comment by Tom-Camp** [comment]
Template changes are include because moving the templates from Branding to Blueprint introduces changes that were previously made and merge in Branding.

File: controls/views.py [hunk 1]
```diff
@@ -400,10 +408,12 @@ def get_context_data(self, **kwargs):
         if self.request.user.has_perm('view_system', system):
             # Retrieve primary system Project
             # Temporarily assume only one project and get first project
-            project = system.projects.first()
+            project = Project.objects.filter(system_id=system.id).first()
```
Comment (woodt) [inline controls/views.py]: Just curious: is this a performance improvement?  It looks functionally equivalent to the line it is replacing.

**Review by woodt — APPROVED** [review]
I think this looks good.  For future reference, it would be preferably to keep formatting changes separate from feature work ... makes it very hard to find the "real" changes in the PR.

File: controls/views.py (hunk 1 above)
Comment (Tom-Camp) [reply controls/views.py:411]: The .first() on line 441 is superfluous and can (should) be removed.

The question of whether querying for a Project by filter on the system id is more performant than assuming that there is only one Project associated with system and getting the first one, I don't know.

Comment (Tom-Camp) [reply controls/views.py]: This change is superfluous and was made when trying to solve a problem that ultimately was solved at the template level. Removing...

**Pull request merged**